hscommon/hsutil/imagebuttonhack.go

```diff
@@ -2,23 +2,19 @@ package hsutil
 
 import (
 	"github.com/ianling/giu"
-	"github.com/ianling/imgui-go"
 )
 
-// MakeImageButton is a hack for giu.ImageButton that manually sets the ImageButton ID by
-// manually setting the ID through imgui
-//nolint:unparam // width and height are always 15 at the time of writing, but may change
+// it looks like childWidget's size must be a bit greater then widget iside of it
+const (
+	stdMod = 8
```

Comment:
what does `stdMod` stand for...?

Reply:
hmm. right, it isn't right name. it comes from `standard modifier`

Reply:
i figured, but the variable is local to this file, you can be more descriptive.

Reply:
da0a135